shell/src/dapps/o-banking/processes/transfer.ts

```diff
@@ -192,65 +192,7 @@ const processDefinition = (processId: string) =>
           onDone: [
             {
               cond: (context) => !!context.data.recipientProfile,
-              target: "#getMaxFlow",
```

Comment:
i don't think this was intended to be in this commit :)

Reply:
That's right :)